libs/src/utils.js

```diff
@@ -39,6 +39,10 @@ class Utils {
     return Web3.utils.isBN(number)
   }
 
+  static toBN (number) {
+    return new Web3.utils.BN(number)
```

Comment:
do we always have a global web3? in the browser context, we don't include it.

Reply:
This is using the imported Web3 from the root web3.js file which gets it from the window. You're right that we can't assume it's there in the browser context, but it's what the client does currently and I made the dashboard do the same thing as well.

Reply:
Coolcool